client/client.go

```diff
@@ -154,7 +157,7 @@ func (c *Client) gethttpRequest(emailid, method string, body bytes.Buffer) (clos
 		respBody := new(bytes.Buffer)
 		_, err := respBody.ReadFrom(resp.Body)
 		if err != nil {
-			return nil, fmt.Errorf("Error : %v",Errors[resp.StatusCode] )
+			return nil, fmt.Errorf("Error : %v",err )
 		}
 		return nil, fmt.Errorf("Error : %v ", Errors[resp.StatusCode])
```

Comment:
Here the Errors array is used to return error. This will again lead to same incomprehensible error. So, Return error message from the response body.

Reply:
Return error from response body here.